Makefile

```diff
@@ -71,6 +71,7 @@ install:
 build:
 	docker-compose build
 
+.ONESHELL:
```

Comment:
Any specific reason for this?
Was there a bug due to the multiline commands?

Reply:
Yes, the env var exported with `export DOCKER_IMAGE_TAG="dev-$$commit-$$timestamp";` was not available in subsequent commands